### ISSUE TITLE
`f`/`F`/`t`/`T` repeat not working since the last commit

### ISSUE THREAD

**Issue by stevenxxiu** [ghostbuster91/nvim-next#24]
Since the last commit <https://github.com/ghostbuster91/nvim-next/commit/50b639da68d9a132bac07c7e99a999a8ed47065f>, repeats with `f`/`F`/`t`/`T` isn't working for me.

My config is:

```lua
opts = function()
  local next_builtins = require('nvim-next.builtins')

  return {
    default_mappings = {
      repeat_style = 'original',
    },
    items = {
      next_builtins.f,
      next_builtins.t,
    },
  }
end
```

Only using `;` in `repeat_last_move()` and only using `,` in `repeat_last_move_opposite()` appears to make things work, but I didn't take a closer look to see what's actually going wrong.

Another issue is that I get the warning:

> nvim-next: map `builtin_T_expr` with `{expr=true}` instead.

**Comment by ghostbuster91** [comment]
> nvim-next: map builtin_T_expr with {expr=true} instead.

This is supposed to appear only if you bind accidentally builtin functions without passing `opts.expr=true`. 

Builtin functions do set this in their definition [here](https://github.com/ghostbuster91/nvim-next/blob/main/lua/nvim-next/builtins/init.lua#L10). 
And this is passed into the `vim.keymap.set` in [here](https://github.com/ghostbuster91/nvim-next/blob/main/lua/nvim-next/init.lua#L30).

Are you sure that you are not binding `f`/`F` etc yourself somewhere else?

**Comment by stevenxxiu** [comment]
I checked and I'm not binding it somewhere else:

```
:nmap
...
n  F           * <Lua 491: ~/.local/share/nvim/lazy/nvim-next/lua/nvim-next/builtins/functions.lua:15>
                 nvim-next: builtin f/F
...
```

I removed all my plugins except *next*, and this still occurs.

**Comment by stevenxxiu** [comment]
Appears to be fixed by #26 indeed.
